helper/utilty.py

```diff
@@ -383,9 +395,10 @@ def upscale_weight(scale, channels, name="weight"):
 
 
 def weight(shape, stddev=0.01, name="weight", uniform=False, initializer="stddev"):
+    # initialize the weights of the graph as a tf variable ( to ensure non-symmetry )
     if initializer == "xavier":
         initial = xavier_cnn_initializer(shape, uniform=uniform)
-    elif initializer == "he":
+    elif initializer == "he": #this is actually the default one because args.py default argument for initializer is "he".
```

Comment:
This function is a utility function. I think these kind of comments should go to model generating function.